tests/resources/functions/dotnet-6.0/Tests.cs

```diff
@@ -1,46 +1,128 @@
 
-    var response = await http.GetStringAsync($"https://jsonplaceholder.typicode.com/todos/{id}");
-    var todo = JsonConvert.DeserializeObject<Dictionary<string, object>>(response, settings: null);
+                foreach (var Entry in Context.Req.Headers)
+                {
+                    Json.Add(Entry.Key, Entry.Value);
+                }
 
-    Console.WriteLine("String1");
-    Console.WriteLine(42);
-    Console.WriteLine(4.2);
-    Console.WriteLine(true);
+                return Context.Res.Json(Json);
+            case "requestBodyPlaintext":
+                return Context.Res.Send((string) Context.Req.Body);
+            case "requestBodyJson":
+                string Key1 = "";
+                string Key2 = "";
 
-    Console.WriteLine("String2");
-    Console.WriteLine("String3");
-    Console.WriteLine("String4");
-    Console.WriteLine("String5");
+                if(Context.Req.Body is string) {
+                    Key1 = "Missing key";
+                    Key2 = "Missing key";
+                } else {
+                    Dictionary<String, Object> Body = (Dictionary<String, Object>) Context.Req.Body;
 
-    return res.Json(new()
-    {
-        { "isTest", true },
-        { "message", "Hello Open Runtimes 👋" },
-        { "header", header },
-        { "variable", varData },
-        { "todo", todo }
-    });
-}
\ No newline at end of file
+                    Key1 = Body.TryGetValue("key1", out var Key1Value) ? Key1Value.ToString() : "Missing key";
+                    Key2 = Body.TryGetValue("key2", out var Key2Value) ? Key2Value.ToString() : "Missing key";
+                }
+
+                return Context.Res.Json(new()
+                {
+                    { "key1", Key1 },
+                    { "key2", Key2 },
+                    { "raw", Context.Req.BodyString }
+                });
+            case "envVars":
+                return Context.Res.Json(new()
+                {
+                    { "var", Environment.GetEnvironmentVariable("CUSTOM_ENV_VAR") ?? null },
+                    { "emptyVar", Environment.GetEnvironmentVariable("NOT_DEFINED_VAR") ?? null },
+                });
+            case "logs":
+                Console.WriteLine("Native log");
+                Context.Log("Debug log");
+                Context.Error("Error log");
+                
+                Context.Log(42);
+                Context.Log(4.2);
+                Context.Log(true);
+
+                Dictionary<string, string> Obj = new Dictionary<string, string>();
+                Obj.Add("objectKey", "objectValue");
+
+                Context.Log(Obj);
+
+                ArrayList Arr = new ArrayList();
+                Arr.Add("arrayValue");
+
+                Context.Log(Arr);
+
+                return Context.Res.Send("");
+            case "library":
+                var response = await http.GetStringAsync($"https://jsonplaceholder.typicode.com/todos/" + Context.Req.BodyString);
```

Comment:
You can use interpolation with `$` strings instead of using + like `$"https...{context.Req.BodyString}"`